deb/common/control

```diff
@@ -92,9 +92,9 @@ Conflicts: rootlesskit
 Replaces: rootlesskit
 Breaks: rootlesskit
 # slirp4netns (>= 0.4.0) is available in Debian since 11 and Ubuntu since 19.10
-Recommends: slirp4netns (>= 0.4.0)
-# Unlike RPM, DEB packages do not contain "Recommends: fuse-overlayfs (>= 0.7.0)" here,
-# because Debian (since 10) and Ubuntu support the kernel-mode rootless overlayfs.
+# fuse-overlayfs is recommended for Debian 10 and 11 (kernel < 5.11). Not needed for Ubuntu.
+Recommends: slirp4netns (>= 0.4.0),
+            fuse-overlayfs (>= 0.7.0)
```

Comment:
This control file is used for both Debian an Ubuntu; can we make this somehow conditional for Debian only? I recall we had some conditional rule for AppArmor on Debian that I removed in https://github.com/docker/docker-ce-packaging/pull/487 - perhaps a similar rule would work for this?

Reply:
Modifying the file with `echo ... >> control` doesn't look robust. So I'd prefer to recommend fuse-overlayfs unconditionally.

Reply:
It's creating a `.substvars` file (not modifying the `control` file)